packages-ts/gauntlet-terra-cw-plus/src/commands/multisig.ts

```diff
@@ -26,9 +26,9 @@ export const wrapCommand = (command) => {
     }
 
     buildCommand = async (flags, args): Promise<TerraCommand> => {
-      if (!AccAddress.validate(process.env.CW3_FLEX_MULTISIG)) throw new Error(`Invalid Multisig wallet address`)
-      if (!AccAddress.validate(process.env.CW4_GROUP)) throw new Error(`Invalid Multisig group address`)
       this.multisig = process.env.CW3_FLEX_MULTISIG as AccAddress
+      if (!AccAddress.validate(this.multisig)) throw new Error(`Invalid Multisig wallet address`)
```

Comment:
Please revert this change. The validation should happen before assigning the variable